eng/SourceBuildPrebuiltBaseline.xml

```diff
@@ -1,5 +1,9 @@
 <UsageData>
   <IgnorePatterns>
-    <UsagePattern IdentityGlob="*/*" />
+    <UsagePattern IdentityGlob="Microsoft.SourceBuild.Intermediate.*/*" />
+
+    <!-- Tracked with https://github.com/microsoft/vstest/issues/4403 -->
+    <UsagePattern IdentityGlob="Microsoft.CodeCoverage.IO/17.7.1-beta.23225.3" />
```

Comment:
Maybe this is no longer needed because of https://github.com/microsoft/vstest/pull/4426

Reply:
Thanks - will update!